.github/workflows/pr-build.yml

```diff
@@ -132,6 +132,12 @@ jobs:
           7z l ./node/release.zip
           ls ${{matrix.platform != 'windows' && '-la' || ''}} ./node/${{ steps.platform.outputs.dir }}/
 
+      - name: Install libOpenCL.so
+        if: matrix.id == 'linux'
+        run: |
+          sudo apt update
+          sudo apt install ocl-icd-opencl-dev
```

Comment:
why do you need `-dev` on CI here?

Reply:
Otherwise, CI can't run go-spacemesh on ubunutu to check the version and everything breaks. :)
```
ERROR: Can not run the Node to check the version: Error: Command failed: /home/runner/work/***/***/node/linux/go-spacemesh version
/home/runner/work/***/***/node/linux/go-spacemesh: error while loading shared libraries: libOpenCL.so.1: cannot open shared object file: No such file or directory
```
https://github.com/spacemeshos/smapp/actions/runs/5071884037/jobs/9110448516

Or do you mean that I'd better install `ocl-icd-opencl` without `-dev` suffix? :)